snap/epoch.go

```diff
@@ -338,7 +338,7 @@ func (z *uint32slice) UnmarshalJSON(bs []byte) error {
 }
 
 func isIncreasing(z []uint32) bool {
-	if len(z) == 0 {
+	if len(z) < 2 {
```

Comment:
does this need a test?

Reply:
this was a small tweak suggested by maciej in a previous pr, which I promised to do in a followup. It's a small tweak that short-circuits a common case at no extra cost, and is covered by tests already, being functionally equivalent to existing code.

Reply:
sorry, I somehow read it yesterday as <= 2 and was confused

Reply:
Thanks for the followup!